scss/os/_utils.scss

```diff
@@ -590,6 +594,11 @@ $somesizes: map-merge((
   top: -1.3rem;
 }
 
+.u-progress__large {
```

Comment:
Should we follow Bootstrap's convention for utilities like these and use `.u-progress-lg`?